frontend/app/src/App.tsx

```diff
@@ -1017,6 +1017,20 @@ export class App extends PureComponent<Props, State> {
 
       // Set the favicon to its default values
       this.onPageIconChanged(`${process.env.PUBLIC_URL}/favicon.png`)
+
+      // Use previously saved layout if exists, otherwise default to CENTERED
+      // Pages using set_page_config(layout=...) will be overriding these values
```

Comment:
I think it would be helpful to have an explanation why this is only used for full app runs. basicially similar to your PR description

Reply:
note: we are having this look up for any reruns now